docs/users/basics/transactions.md

```diff
@@ -36,7 +36,53 @@ Explain that transactions can interoperate with other blockchains.
 
 ### Ethereum Transactions
 
-<!-- TODO: transactions that interact with the EVM -->
+Evmos supports the following Ethereum transactions.
```

Comment:
I think this is useful but imo it's not targeted at the Basics section (new users). This should maybe go under the Developers section.